js/part7.js

```diff
@@ -0,0 +1,22 @@
+`use strict`;
+
+let $oneCategoryDiscont = 0.03;
+let $twoCategoryDiscont = 0.05;
+let $threeCategoryDiscont = 0.07;
+
+let $amount = prompt(`Розрахунок суми до оплати зі знижкою. Введіть початкову суму у гривнях.`);
+
+if ($amount === `` || +$amount <= 0 || isNaN($amount)) {
+    alert(`Ви впевнені? Спробуйте ще раз.`);
+} else if (+$amount < 200) {
+    alert(`До оплати ${+$amount} (знижки немає)`);
+} else if (+$amount >= 200 && +$amount < 300) {
+    alert(`До оплати ${+$amount - (+$amount * $oneCategoryDiscont)} (знижка ${$oneCategoryDiscont * 100}%)`);
```

Comment:
You can simplify calculations
Since you have discounts already (0.03, 0.05, 0.07) you can just do `$amount * (1 - oneCategoryDiscont)`

Or even simper
`let $oneCategoryToDiscount = 0.97;`
and then
`$amount * $oneCategoryToDiscount`